source/stylesheets/mpv/_bootstrap-changes.sass

```
@@ -35,6 +35,7 @@ a:hover .big-circle, a:hover .circle
 // blockquotes that change the font size. Override them with inherit.
 blockquote
   margin: 0
+  font-size: inherit
   p
     font-size: inherit
     font-weight: inherit
```

i did some experimenting with the browser inspector and I think the problematic rules merely moved:
```suggestion
  font-size: inherit
  font-weight: inherit
  p
```

---

Changed